### PR TITLE
Withdraw function interface change

### PR DESCRIPTION
**Pull request by lazarmitic**

### CODE REVIEW

File: packages/tempus-client_v2/src/components/withdraw/Withdraw.tsx [hunk 1]
```diff
@@ -218,7 +198,8 @@ const Withdraw: FC<WithdrawOutProps> = ({ onWithdraw }) => {
         lpTokenAmountParsed,
         minPrincipalsStaked,
         minYieldsStaked,
-        minRate,
+        estimatedWithdrawData.yieldsRate,
+        ethers.utils.parseEther(actualSlippage),
```
Comment (danieledamiani) [inline packages/tempus-client_v2/src/components/withdraw/Withdraw.tsx:202]: please use `parseUnit`

**Review by danieledamiani — CHANGES_REQUESTED** [review]
a comment

File: packages/tempus-client_v2/src/components/withdraw/Withdraw.tsx (hunk 1 above)
Comment (lazarmitic) [reply packages/tempus-client_v2/src/components/withdraw/Withdraw.tsx:202]: It's hardcoded on contract side to use 18 decimals for slippage, slippage is not a token, so no need to use token precision here.

Comment (danieledamiani) [reply packages/tempus-client_v2/src/components/withdraw/Withdraw.tsx:202]: then for consistency let's do `parseUnit(actualSlippage, 18)`

Comment (lazarmitic) [reply packages/tempus-client_v2/src/components/withdraw/Withdraw.tsx]: fixed

**Review by danieledamiani — APPROVED** [review]

**Pull request merged**